### DISCUSSION TITLE
what is dynamic interfaces  in client-side rendering?

### DISCUSSION THREAD

**Discussion by poemtalent** [nuxt/nuxt#16778] [Questions]
Is anyone know what dynamic interfaces is in client-side rendering? I saw it in documentation
![image](https://user-images.githubusercontent.com/48041365/185581067-0af5897e-74e1-4358-9b22-f7bccd825135.png)

**Comment by sevillaarvin** [comment]
It means the User Interface is able to change without doing a page refresh.
